package.json

```diff
@@ -21,7 +21,8 @@
     "test:features": "cross-env CACHE=true FEATURES=true karma start config/karma/karma.config.features.js",
     "test:playground": "karma start config/karma/karma.config.transformer.playground.js",
     "test:playground:build": "karma start config/karma/karma.config.transformer.playground.build.js",
-    "release": "cp -r package.json package-lock.json README.md dist && semantic-release",
+    "package": "cp -r package.json package-lock.json README.md dist",
```

Comment:
I would like to understand a bit more the need of a package script. We are currently copying these files only for the release. If you were renaming package script to copyPackageAndReadmeToDist I would be okay.

Reply:
What about a scope similar to tests: `dist:package` and/or `dist:release` 🤔 ?

Reply:
I'm okay with dist:release :)

Reply:
@martinjlowm Sorry there was a misunderstanding. I think release can stay there. I would do write instead

```
"dist:release": "cp -r package.json package-lock.json README.md dist",
"release": "npm run dist:release && semantic-release",
```

Reply:
Morning :) Doneso!

Reply:
completely forgot about this. I'm okay with collect.. I'll merge it :)